.pairs

```diff
@@ -1,17 +1,27 @@
 # .pairs - configuration for 'git pair'
 pairs:
   # <initials>: <Firstname> <Lastname>[; <email-id>]
-  jh: Jenny Heath; jheath
-  bg: Ben Golder; bgolder
-  td: Tom Dooner; tdooner
-  lk: Laura Kogler; lkogler
-  mtm: Molly T-M; mollyt
-  jg: Jonathan Greenberg; jgreenberg
-  bv: Ben Vandgrift; bvandgrift
-  dbd: Dimitri DeFigueiredo; ddefigueiredo
-  al: Asheesh Laroia; alaroia
-email:
-  prefix: pair
-  domain: codeforamerica.org
-  no_solo_prefix: true
+  jh: Jenny Heath
+  bg: Ben Golder
+  td: Tom Dooner
+  lk: Laura Kogler
+  mtm: Molly T-M
+  jg: Jonathan Greenberg
+  bv: Ben Vandgrift
+  dbd: Dimitri DeFigueiredo
+  al: Asheesh Laroia
+  sn: Sarah Niemeyer
+  as: Alex Sartan
+email_addresses:
```

Comment:
i imagine there's some documentation i haven't read that indicates `email` -> `email_addresses` works - if you were able to duet-commit then i think this looks good!